### PR TITLE
USHIFT-333: add ingress-to-route-controller RBAC

### PR DESCRIPTION
**Pull request by dhellmann**
Add assets for RBAC for ingress-to-route-controller.
Update rebase.sh to pull updates to those assets in the future.

Partial fix for [USHIFT-333](https://issues.redhat.com//browse/USHIFT-333)
Partial fix for #860

### CODE REVIEW

**Comment by dhellmann** [comment]
I put a hold on this, since it still includes some debug statements that we won't want to merge.

@benluddy let me know if you think this is useful on its own or if you want to just pull the useful parts into a PR of your own.

**Review by benluddy — COMMENTED** [review]
@dhellmann I would support merging this PR today if you remove the changes to vendor (and I have the one comment on rebase.sh).

File: scripts/rebase.sh [hunk 1]
```diff
@@ -455,6 +455,9 @@ update_manifests() {
     #    Replace all openshift-router operand manifests
     rm -f "${REPOROOT}"/assets/components/openshift-router/*
     cp "${STAGING_DIR}"/cluster-ingress-operator/assets/router/* "${REPOROOT}"/assets/components/openshift-router 2>/dev/null || true
+    # Copy embedded manifests applied by the COCMO rather than CVO
+    cp "${STAGING_DIR}"/cluster-openshift-controller-manager-operator/bindata/v3.11.0/openshift-controller-manager/ingress-to-route-controller-clusterrole.yaml "${REPOROOT}"/assets/components/openshift-router 2>/dev/null || true
```
[the remark this answers is not in the document]
Comment (dhellmann) [reply scripts/rebase.sh:459]: Yeah, I was following the established pattern there. I want to revisit that independently of this PR.

**Comment by dhellmann** [comment]
> @dhellmann I would support merging this PR today if you remove the changes to vendor (and I have the one comment on rebase.sh).

I have removed that commit.

**Pull request merged**